x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -446,6 +446,7 @@ export interface AgentUpgradeDetails {
   metadata?: {
     scheduled_at?: string;
     download_percent?: number;
+    download_rate?: number; // bytes per second
```

Comment:
Should we update elasticsearch mapping to add this too?  https://github.com/elastic/elasticsearch/blob/bb5bd7f4d72c2a28bf2e84539199dbab137b243e/x-pack/plugin/core/template-resources/src/main/resources/fleet-agents.json#L248

Reply:
and fleet server too https://github.com/elastic/fleet-server/blob/4d155c31683f2afef333f087061fc09b6311805e/model/openapi.yml#L282

Reply:
Good catch, I'll update

Reply:
Raised pr: https://github.com/elastic/elasticsearch/pull/103566